### DISCUSSION TITLE
Remove certain app-indicators from tray

### DISCUSSION THREAD

**Discussion by codelif** [Alexays/Waybar#2449] [Q&A]
Is there anyway to remove certain app-indicators from tray?

I know it sounds silly, but I have been using nm-applet as a network strength indicator in conjunction with a network module like this.

![20230901_10h21m19s_grim](https://github.com/Alexays/Waybar/assets/68972644/0194d011-022f-4507-8b6a-69720aa11d1e)

But I have to open another app with an indicator and it totally messes up the look. It does not have an option to disable the Indicator. Though it's not the end of the world or anything but I would still like if there was a way. (In this case, I could just use the web app but I still want to know if there is a way)

I have set reverse-direction to true, so that helps ig.

![20230901_13h24m48s_grim](https://github.com/Alexays/Waybar/assets/68972644/79d31bdb-f1c2-4132-a227-58e6c2e9900c)

Also excuse the Wi-Fi SSID, I didn't know any better!!

**Comment by guttermonk** [comment]
I agree, it would be nice if there was a way to separate out the `blueman-applet` and the `network-manager-applet` from `tray` to give them a specific color and location.

**Comment by guttermonk** [comment]
There is an open feature request for this: https://github.com/Alexays/Waybar/issues/2842

Please give it a thumbs up and leave a comment if you'd like to see this added.
